tests/testthat/test-label-dollar.R

```diff
@@ -25,15 +25,15 @@ test_that("rescale_large works", {
   x <- 10^(seq(0, 18, 3))
 
   expect_equal(
-    label_dollar(rescale_large = rescale_short_scale())(x),
+    label_dollar(scale_cut = cut_short_scale())(x),
     c("$1", "$1K", "$1M", "$1B", "$1T", "$1,000T", "$1,000,000T")
   )
   expect_equal(
-    label_dollar(rescale_large = rescale_long_scale())(x),
+    label_dollar(scale_cut = cut_long_scale())(x),
     c("$1", "$1K", "$1M", "$1,000M", "$1B", "$1,000B", "$1T")
   )
   expect_equal(
-    label_dollar(rescale_large = c(k = 3L, m = 6L, bn = 9L, tn = 12L))(x),
+    label_dollar(scale_cut = c(k = 1e3, m = 1e6, bn = 1e9, tn = 1e12))(x),
     c("$1", "$1k", "$1m", "$1bn", "$1tn", "$1,000tn", "$1,000,000tn")
   )
 })
```

Comment:
It might also be good to have a test when the currency symbol is a suffix (e.g. France). It seems like an edge case we should be checking for regressions.